packages/cli/lib/services/model.service.ts

```diff
@@ -139,6 +139,20 @@ class ModelService {
             if (hasUndefined) {
                 tsType = `${tsType} | undefined`;
             }
+
+            if (tsType.includes('|')) {
```

Comment:
Does supporting union types make the specific handling of undefined and null above unecessary?

Reply:
I think my earlier comment clarifies this -- but let me know if not!